src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -1009,9 +1009,6 @@ void uct_rc_mlx5_iface_common_query(uct_ib_iface_t *ib_iface,
     uct_ib_device_t *dev = uct_ib_iface_device(ib_iface);
 
     /* Atomics */
-    iface_attr->cap.flags        |= UCT_IFACE_FLAG_ERRHANDLE_ZCOPY_BUF |
-                                    UCT_IFACE_FLAG_ERRHANDLE_REMOTE_MEM;
```

Comment:
i think that given recent changes in the PR, we don't have to delete these since we always call the handler

Reply:
@yosefe right but we don't print the error in this case (it's diag). the test expects a printed error.

Reply:
the test can check for ucs_diag message, or config failure level to error